src/App.js

```diff
@@ -1,101 +1,129 @@
-import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
 import { useState } from 'react'
 
-import Navbar from "./Components/Navbar"
-import Timer from "./Components/Timer"
-import AddTimer from "./Components/AddTimer"
-import ManageTimers from "./Components/ManageTimers"
-import './App.css';
+import Navbar from './Components/Navbar'
+import Timers from './Components/Timers'
+import AddTimer from './Components/AddTimer'
+import ManageTimers from './Components/ManageTimers'
+import './App.css'
+
+// Defining initial state outside of the component can make the component easier to read.
+const timersInitialState = [
```

Comment:
put constant in a constant file

Reply:
I disagree that a constant file is necessary, but somewhere down the line in @adrianno33's learning they will probably start keeping data like this in a database rather than in state.